src/commands/config.ts

```
@@ -10,32 +10,19 @@
-	EmbedBuilder,
-	Locale,
-	PermissionFlagsBits,
-	SlashCommandBuilder,
-	type StringSelectMenuInteraction,
-} from "discord.js";
+import * as Djs from "discord.js";
 import { getUl, ln, t } from "../i18n";
-import { CommandName, type Translation } from "../interface";
-import { getConfig, optionMaps, setConfig } from "../maps";
-import { logInDev } from "../utils";
+import {ConfigurationKey, ConfigurationKeys, Translation} from "../interface";
```

Biome (2.1.2)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)